README.md

```diff
@@ -88,6 +88,9 @@ To run tests, run the following command:
  npx hint .
  npx stylelint "**/*.{css,scss}"
 ```
+### Deployment
+Woow, we are live 🚀
+<p>Click <a href="https://fredrickotabil.github.io/Personal_Portfolio/" target="_blank">here</a> to view my website</p>
 
```

Comment:
- [ ] [OPTIONAL] There is a section in the README template reserved for the live version link. I think it would be great if you put the link under there. :+1: 

![image](https://user-images.githubusercontent.com/17243325/223423908-b8b39986-ae2e-49b1-8a09-fc81be9cd26f.png)